packages/components/breadcrumbs/src/breadcrumbs.tsx

```diff
@@ -14,7 +14,11 @@ const Breadcrumbs = forwardRef<"div", BreadcrumbsProps>((props, ref) => {
     children,
     childCount,
     itemProps,
-    separator = <ChevronRightIcon />,
+    separator = (
+      <div className="rtl:transform rtl:rotate-180">
```

Comment:
@mrbadri all the tailwindcss styles should be in the `theme` package, this is because the users load the styles only from that folder https://nextui.org/docs/guide/installation#tailwind-css-setup so all tailwindcss classes out of there will not be compiled